File: .sdk/tm/php/utility/PrepareBody.php
<?php
declare(strict_types=1);

// BluefinShieldconex SDK utility: prepare_body

class BluefinShieldconexPrepareBody
{
    public static function call(BluefinShieldconexContext $ctx): mixed
    {
        if ($ctx->op->input === 'data') {
            return ($ctx->utility->transform_request)($ctx);
        }
        return null;
    }
}

File: java/utility/PrepareBody.java
package voxgig.bluefinshieldconexsdk.utility;

import voxgig.bluefinshieldconexsdk.core.Context;

final class PrepareBody {

  private PrepareBody() {}

  static Object prepareBody(Context ctx) {
    if ("data".equals(ctx.op.input)) {
      return ctx.utility.transformRequest.apply(ctx);
    }

    return null;
  }
}
